DemoqaProject/tests/AlertsTest.cs

```diff
@@ -0,0 +1,18 @@
+using DemoqaProject.PageObjects;
+using NUnit.Framework;
+
+namespace DemoqaProject
+{
+    public class AlertsTest : BaseSeleniumTest
+    {
+        [Test]
+        public void CheckAlertsText()
+        {
+            String getHeader = Config.alertsText;
```

Comment:
You can inline this into Assert.AreEqual, no need to create a variable as you don't use it several times.

Reply:
done